ios/FluentUI/Extensions/UIApplication+Extensions.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+extension UIApplication {
```

Comment:
It seems like this extension is not being used anymore. Can you delete this file?